Automattic/wp-calypso
# FSE: Avoid breaking sites even if Jetpack is unavailable

Fixes #79956, #77573.



Related to #

## Proposed Changes

* Does not load Help Center if not on WPCOM and Jetpack is not active.
* Bails Help Center early if `wpcom_get_site_purchases()` is not available to avoid breaking sites where the function is not available.
* Replaces a reference of Jetpack_Options with a direct options call to avoid breaking sites where the class is not available.
* Updates plugin readme to better reflect the experimental nature of the plugin.

## Testing Instructions



* Follow the instructions under "ETK and Atomic Sites" (PCYsg-ly5-p2) to download the zip build of ETK
* Upload the plugin on a self-hosted site and activate—it should not fatal.
* Upload the plugin on a WoA site and deactivate Jetpack through `_cli`. The site should not fatal.

## Pre-merge Checklist
